docs/evaluate/temporal-cloud/support.mdx

```diff
@@ -1,62 +1,72 @@
 
-Temporal endeavors to ensure you are successful with Temporal Cloud. We offer explicit guarantees for support.
+Temporal endeavors to ensure you are successful with Temporal Cloud.
+We offer explicit guarantees for support.
 Temporal Cloud customers get break/fix support with an agreed-upon set of SLAs for prioritized issues.
 We use a ticketing system for entering, tracking, and closing these issues.
 
 If an issue occurs, the team also provides support through a dedicated Slack channel, forums, and a knowledge base.
 We offer two levels of support defined by their availability and SLAs in the following table:
 
-|                          | **Basic**                                                                                      | **Premium**                                                                               |
-| ------------------------ | ---------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| **Response times**       | P0: 1 business hour<br />P1: 4 business hours<br />P2: 1 business day<br />P3: 2 business days | P0: 30 minutes<br />P1: 1 business hour<br />P2: 4 business hours<br />P3: 1 business day |
-| **Onboarding & Success** | Development, deployment, and optimization guidance                                             | Development, deployment, and optimization guidance                                        |
-| **Channels**             | Community Slack, Email, Forums, Documentation, Knowledge Base                                  | Private Slack Channel, Email, Forums, Documentation, Knowledge Base                       |
+| |Essentials | Business | Enterprise | Mission Critical |
+|-|------------|----------|------------|------------------|
+| **Availability**<br />(Based on<br />Time-zones) | **P0–3**: 9–5 Mon–Fri | **P0–3**: 9–5 Mon–Fri | **P0**: 24×7, <br />(On Page Service)<br /> **P1–3**: 9–5 Mon–Fri | **P0**: 24×7 (On Page Service)<br />**P1**: 9-5, 7 days/week<br />**P2–3**: Mon–Fri |
+| **Response Time** | **P0**: 1 business day<br />**P1**: 1 business day<br />**P2**: 1 business day<br />**P3**: 2 business days | **P0**: 2 business hours<br />**P1**: 2 business hours<br />**P2**: 1 business day<br />**P3**: 2 business days | **P0**: 30 minutes<br />**P1**: 1 business hour<br />**P2**: 4 business hours<br />**P3**: 1 business day | **P0**: 15 minutes<br />**P1**: 1 business hour<br />**P2**: 4 business hours<br />**P3**: 1 business day |
+| **DSE** | - | - | Add-on | DSE Included (1 Unit) |
+| **Channels** | Community<br/>Temporal Support Portal | Community<br />Temporal Support Portal | Community<br />Temporal Support Portal<br />Private Slack | Community<br />Temporal Support Portal<br />Private Slack |
 
-**Business Hours**: 05:00-17:00 Mon-Fri US Pacific Time.
-For P0 issues, coverage is 24×7.
+:::info BUSINESS HOURS AND TIMEZONES
```

Comment:
another capitalization callout

Reply:
This is automatically capped in rendering. Putting it in caps in the mdx reflects how it will be rendered.

Reply:
Lower casing these and pushing.